src/components/Footer.js

```diff
@@ -0,0 +1,23 @@
+
+import { Navbar } from 'reactstrap';
```

Comment:
when importing multiple components from the same location encapsulate them within parenthesis and separate with a comma
`import { Nav, Navbar } from 'reactstrap'`